forms-flow-web/src/components/Form/EditForm/FormEdit.js

```diff
@@ -883,6 +883,8 @@ const handleSaveLayout = () => {
       newFormData.componentChanged = true;
       newFormData.newVersion = true;
       newFormData.parentFormId = previousData.parentFormId;
+      newFormData.title = previousData.formName;
```

Comment:
this is correct but , previous data is using for checking any changes happened in the mapper earlier code, could you please change the `current mapper variable  (processListData)  or formData`, the above line also